CONTRIBUTING.md

```diff
@@ -26,11 +26,15 @@ Install the following dependencies (Instructions are linked for each dependency)
 - [kubectl](https://kubernetes.io/docs/tasks/tools/#kubectl)
 - [LLVM](https://apt.llvm.org/)
 
+For the debian based distributions:
+
     ```console
     $ sudo apt-get update
 
-    $ sudo apt-get install make clang llvm libbpf-dev libelf-dev
-     ```
+    $ sudo apt-get install make zlib1g libclang-14-dev llvm-14-dev libbpf-dev libelf-dev
```

Comment:
Sorry, just remembered, we also need `pkg-config` to build libbpf as well as a C compiler to compile it

Reply:
Good point. Mentioning it separately will be nice.